### ISSUE TITLE
Helm push lose requirments.yaml file

### ISSUE THREAD

**Issue by yangzb09** [chartmuseum/helm-push#52]
HELM version
Client: &version.Version{SemVer:"v2.16.1", GitCommit:"bbdfe5e7803a12bbdf97e94cd847859890cf4050", GitTreeState:"clean"}
HELM push version: 0.8.0
After upgrade to helm push 0.8.0, The Charts pushed to chartmuseum, lose requirments.yaml. 
```
dependencies:
- name: kube-state-metrics
  version: 2.2.1
  repository: https://kubernetes-charts.storage.googleapis.com
  condition: kube-state-metrics.enabled
- name: A
  version: 1.0.1 
  repository: "file://../A"
  condition: A.enabled
```
Steps to push
```
helm repo add chartmuseum https://charts.software.io
helm dep up software
helm dep build software
helm push software chartmuseum -v ">0.0.0-0" -f 
```
Steps to download
```
wget https://charts.software.io/charts/software-\>0.0.0-0.tgz 
tar zxvf software-\>0.0.0-0.tgz  -C ./dir
s dir
Chart.yaml  charts      templates   values.yaml
```
requirments.yaml and lock are lose.

Anyway, downgrade to 0.7.1 can workaround.

**Comment by jdolitsky** [comment]
@yangzb09 yes this would make sense.. thanks for opening. 

Helm 3 has moved requirements.yaml dependencies into Chart.yaml. The problem code is here: https://github.com/chartmuseum/helm-push/blob/master/pkg/helm/chart.go#L33 (`chartutil.Save`)

I can work on using the old Helm 2 chartutil.Save for charts w/ apiVersion: 1

As a workaround, you should be able to first package the directory then push it:
```
helm package software/
helm push software*.tgz chartmuseum -v ">0.0.0-0" -f 
```

**Comment by jdolitsky** [comment]
@yangzb09 this has been fixed and a new release available: `v0.8.1`

Can you please verify on your end?
